WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmountViewModel.swift

```diff
@@ -35,17 +35,17 @@ final class SimplePaymentsAmountViewModel: ObservableObject {
     ///
     var onOrderCreated: (Order) -> Void = { _ in }
 
-    /// Returns true when amount has less than two characters.
-    /// Less than two, because `$` should be the first character.
+    /// Returns true when amount is a valid number bigger than .zero.
```

Comment:
I think this comment is reversed (the done button should be disabled/this returns true when the amount is _not_ a valid number bigger than .zero).

Reply:
🤦